app/scripts/admin/controllers/hiv-monthly-summary-indicators.controller.js

```diff
@@ -81,7 +82,12 @@
       if ($scope.countBy && $scope.countBy !== '' && $scope.reportName && $scope.reportName!=='' && $scope.startDate
         && $scope.startDate!=='' && $scope.selectedIndicatorTags.indicatorTags &&
         $scope.selectedIndicatorTags.indicatorTags !== [])
-        var locations =getSelectedLocations($scope.selectedLocations);
+        var locations = '';
+      if ($scope.selectedLocation !== '') {
+        locations = $scope.selectedLocation;
```

Comment:
Is not possible for $scope.selectedLocation to be null or undefined in which case if condition would be true and I am sure that is not what you want.